data/committees.js

```diff
@@ -8,7 +8,7 @@ export default [
 		class: 'studio',
 		slug: 'studio',
 		tagline: 'Creative Expression through Games',
-		external_link: null,
+		external_link: 'https://studio.uclaacm.com/',
 		image: '/images/committees/studio_wordmark.svg',
 		wordmark_dark: '/images/committees/studio_wordmark_dark.svg',
 		diamond: '/images/diamonds/studio_diamond.svg',
```

Comment:
can we also add studio's new website (https://studio.uclaacm.com) in the links section here? thanks